hypha/templates/base-apply.html

```diff
@@ -42,11 +42,6 @@
         <script src="{% static 'js/js.cookie.min.js' %}"></script>
         <script src="{% static 'js/main-top.js' %}"></script>
 
-        <!-- alphine js start-->
-        <script defer src="{% static 'js/apply/vendor/alpine-3.11.1.min.js' %}"></script>
-        <style> [x-cloak] {display: none !important} </style>
-        <!-- apline.js ends-->
-
```

Comment:
This should not be removed I think?

Reply:
You're right! not sure why I thought `base_apply.html` was inherited from the `base.html`.

Reply:
Updated.